extension/modules/service.js

```diff
@@ -1363,7 +1363,7 @@ var PersonaService = {
         if (uri.host != this.addonsHost)
             return;
 
-        if (endsWith(uri.filePath, "/users/logout") || aRequest.requestMethod == "POST" && endsWith(uri.filePath, "/users/login"))
+        if (endsWith(uri.filePath, "/users/logout") || aRequest.requestMethod == "POST" && endsWith(uri.filePath, "/users/login") || endsWith(uri.filePath, "/api/v3/accounts/authenticate/"))
```

Comment:
That authenticate is a GET request. If user validates, it is redirected to AMO. It happens after the POST request.

```
POST XHR https://api.accounts.firefox.com/v1/account/login [HTTP/1.1 200 OK 446ms]
POST XHR https://api.accounts.firefox.com/v1/certificate/sign [HTTP/1.1 200 OK 264ms]
POST XHR https://oauth.accounts.firefox.com/v1/authorization [HTTP/1.1 200 OK 338ms]
POST XHR https://accounts.firefox.com/metrics [HTTP/1.1 200 OK 270ms]
**GET https://addons.mozilla.org/api/v3/accounts/authenticate/ [HTTP/1.1 302 FOUND 436ms]**
```